src/renderers/PartialViewRenderers/region/facilitiesInfo.js

```diff
@@ -99,6 +102,20 @@ function getFacilitiesInfo() {
   $('#btnCloseAllFacilities').hide();
 }
 
+function sortFacility(facility) {
+  facility.sort((facilityA, facilityB) => {
+    const NameA = facilityA.facilityName.toUpperCase();
+    const NameB = facilityB.facilityName.toUpperCase();
+    if (NameA < NameB) {
+      return -1;
+    } if (NameA > NameB) {
+      return 1;
+    }
+    return 0;
```

Comment:
And what do these return numbers mean?

Reply:
Its the jasacript array sorting function that allows costum parameters. In this case, we pass a function where if the object property of facilityName in A is bigger than B, its wrong and needs to be switches since its A > B. if the reverse is true, it returns -1, meaning it doenst need to be sorted. else it returns 0 which just means it also doesnt need to be sorted since its the same.

Reply:
I think these numbers mean the order of the item, where I think 1 is higher, -1 is lower and 0 is equal

Reply:
Wait, that means you can just have `If NameA > NameB -> return 1, else return 0` right?

Reply:
Nah the sorting function still needs to know if its bigger smaller or the same. altho thats true i can simplify the if by using a shorthand.

Reply:
nvm it looks horrible with the if shorthand WKWKWK